src/components/TransactionOverview/TransactionsOverview.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import transactions from "../../services/api/transactions.json";
+import { StyledOverview } from "./TransactionsOverview.styled";
+
+const getTotal = (type) => {
```

Comment:
rename it to `getTotalAmount` and create object to hold types

```
const TRANSACTIONS_TYPES = {
   expense: 'expense',
   income: 'income',
}
```

and then call it like that `getTotalAmount(TRANSACTIONS_TYPES.expense)`

Reply:
done